modules/ROOT/pages/clauses/set.adoc

```diff
@@ -498,6 +536,31 @@ The newly-labeled node is returned by the query.
 Labels added: 1
 |===
 
+[role=label--new-5.24]
+[[set-set-a-dynamic-label-on-a-node]]
+== Dynamically setting a label
+
+`SET` can be used to set a label on a node even when the label is not statically known.
+
+.Query
+[source, cypher]
+----
+MATCH (n:Swedish)
+SET n:$(n.name)
```

Comment:
Can we have an example where the label to add comes from a Cypher parameter, what would the syntax look like ? 

```
SET n:$($label) ?
``` 

Same for multiple labels.